File: Serene/Serene.Script/Imports/ServerImports/Northwind.CategoryLangRow.cs
using jQueryApi;
using Serenity;
using Serenity.ComponentModel;
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Serene.Northwind
{
    [Imported, Serializable, PreserveMemberCase]
    public partial class CategoryLangRow
    {
        [InlineConstant] public const string IdProperty = "Id";
        [InlineConstant] public const string NameProperty = "CategoryName";
        [InlineConstant] public const string LocalTextPrefix = "Northwind.CategoryLang";

        public Int32? Id { get; set; }
        public Int32? CategoryId { get; set; }
        public Int32? LanguageId { get; set; }
        public String CategoryName { get; set; }
        public String Description { get; set; }

        [Imported, PreserveMemberCase]
        public static class Fields
        {
            [InlineConstant] public const string Id = "Id";
            [InlineConstant] public const string CategoryId = "CategoryId";
            [InlineConstant] public const string LanguageId = "LanguageId";
            [InlineConstant] public const string CategoryName = "CategoryName";
            [InlineConstant] public const string Description = "Description";
        }
    }
}

File: Serene/Serene.Web/Imports/ServerTypings/Northwind.CategoryLangRow.ts
namespace Serene.Northwind {
    export interface CategoryLangRow {
        Id?: number;
        CategoryId?: number;
        LanguageId?: number;
        CategoryName?: string;
        Description?: string;
    }

    export namespace CategoryLangRow {
        export const idProperty = 'Id';
        export const nameProperty = 'CategoryName';
        export const localTextPrefix = 'Northwind.CategoryLang';

        export namespace Fields {
            export declare const Id: string;
            export declare const CategoryId: string;
            export declare const LanguageId: string;
            export declare const CategoryName: string;
            export declare const Description: string;
        }

        [
            'Id', 
            'CategoryId', 
            'LanguageId', 
            'CategoryName', 
            'Description'
        ].forEach(x => (<any>Fields)[x] = x);
    }
}

